web/src/scss/variables.scss

```diff
@@ -6,7 +6,12 @@ $sub-nav-text-color: #9c9c9c;
 
 // colors
 $primary-color: #326de6;
+$primary-dark-color: darken($primary-color, 8%);
```

Comment:
we can still use these sass functions but we should probably only assign their values to variables - rather than using them directly in the selectors